docs/docsite/rst/getting_started/get_started_inventory.rst

```diff
@@ -38,6 +38,8 @@ Continue getting started with Ansible and build an inventory as follows:
    .. note::
       Pass the ``-u`` option with the ``ansible`` command if the username is different on the control node and the managed node(s).
 
+      Pass the ``-k`` option if SSH authentication on the managed node(s) requires entering a password.
```

Comment:
```suggestion
      Pass the ``-k`` option if SSH authentication on the managed node(s) requires a password.
```

Reply:
was thinking of it:) Committed, thanks, PTAL